lib/rspec/core/memoized_helpers.rb

```diff
@@ -285,6 +288,9 @@ def let(name, &block)
           # We have to pass the block directly to `define_method` to
           # allow it to use method constructs like `super` and `return`.
           raise "#let or #subject called without a block" if block.nil?
+          raise(
+            "#let or #subject called with a reserved name ##{name}"
+          ) if ReservedKeywords.include? name.to_s
```

Comment:
Instead of always calling `to_s` on the `name`, can we just make the `RESERVED_METHODS` be a list of symbols?

And mine using parens on the `include?` call?  Since it returns a value we act upon it's kinda weird (and non-standard) to avoid parens there.